legion/legion/edi/deploy.py

```diff
@@ -219,10 +219,10 @@ def undeploy_kubernetes(args):
     """
     edi_client = legion.external.edi.build_client(args)
     try:
-        edi_client.undeploy(args.model_id, args.grace_period)
+        edi_client.undeploy(args.model_id, args.grace_period, args.model_version)
     except Exception as exception:
-        if 'Cannot find deployment' in str(exception) and args.ignore_not_found:
-            print('Cannot find deployment - ignoring')
+        if 'No one model can be found' in str(exception) and args.ignore_not_found:
```

Comment:
Oh, this just crap.

Reply:
Yeah, we should replace current exception system in future

Reply:
OK, will replace here